### ISSUE TITLE
AttributeError: module 's3prl' has no attribute 'hub'

### ISSUE THREAD

**Issue by ashishBT** [s3prl/s3prl#206]
I am trying to run the latest version of s3prl. To do this, I have executed the following cmd:

```sh
git clone https://github.com/s3prl/s3prl.git
cd s3prl
pip install -e ./
```

then 

```sh
cd s3prl
python3 run_downstream.py -m train -d asr -n AUDIO_ALBERT -u audio_albert 
```

But I am getting following error:
```python
################################################################################
### WARNING, path does not exist: KALDI_ROOT=/mnt/matylda5/iveselyk/Tools/kaldi-trunk
###          (please add 'export KALDI_ROOT=<your_path>' in your $HOME/.profile)
###          (or run as: KALDI_ROOT=<your_path> python <your_script>.py)
################################################################################

[hubconf] can not import .upstream.decoar2.hubconf: No module named 'fairseq'... Pass.
[hubconf] can not import .upstream.hubert.hubconf: No module named 'fairseq'... Pass.
[hubconf] can not import .upstream.roberta.hubconf: No module named 'fairseq'... Pass.
[hubconf] can not import .upstream.pase.hubconf: No module named 'pase'... Pass.
Traceback (most recent call last):
  File "run_downstream.py", line 15, in <module>
    from s3prl import hub
  File "/home/users/Projects/s3prl/s3prl/hub.py", line 12, in <module>
    _module = _importlib.import_module(_module_name, package=__package__)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/users/Projects/s3prl/s3prl/upstream/mos_prediction/hubconf.py", line 4, in <module>
    from .expert import UpstreamExpert as _UpstreamExpert
  File "/home/users/Projects/s3prl/s3prl/upstream/mos_prediction/expert.py", line 6, in <module>
    import s3prl.hub as hub
AttributeError: module 's3prl' has no attribute 'hub'
```
To debug the error, I listed the installed modules using `pip list`

| Package | Version | Location |
| --- | --- | --- |
| s3prl | 0.3.1 | /home/users/Projects/s3prl |

I think Location should be */home/users/Projects/s3prl/s3prl* insted of */home/users/Projects/s3prl*
Could you please help me to resolve the aforementioned error?

**Comment by leo19941227** [comment]
Hey according to the first line of error. I think it is because you not yet install the fairseq. Please follow the instructions on the main README.md!

**Comment by ashishBT** [comment]
Hello, I have tried to install fairseq using `pip install fairseq@git+https://github.com//pytorch/fairseq.git@f2146bdc7abf293186de9449bfa2272775e39e1d#egg=fairseq`
but got some error, so finally installed using 
```sh
git clone https://github.com/pytorch/fairseq.git
cd fairseq/
git checkout f2146bdc7abf293186de9449bfa2272775e39e1d
python setup.py install
```
Afterthat, I have also installed the Upstream pase

Even after installing all the dependencies, while executing `python3 run_downstream.py -m train -d asr -n AUDIO_ALBERT -u audio_alber` I am getting following error:
```python
Traceback (most recent call last):
  File "run_downstream.py", line 15, in <module>
    from s3prl import hub
  File "/home/users/Projects/s3prl/s3prl/hub.py", line 12, in <module>
    _module = _importlib.import_module(_module_name, package=__package__)
  File "/usr/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/users/Projects/s3prl/s3prl/upstream/mos_prediction/hubconf.py", line 4, in <module>
    from .expert import UpstreamExpert as _UpstreamExpert
  File "/home/users/Projects/s3prl/s3prl/upstream/mos_prediction/expert.py", line 6, in <module>
    import s3prl.hub as hub
AttributeError: module 's3prl' has no attribute 'hub'
```

**Comment by leo19941227** [comment]
Thanks for reporting!
I am fixing this.

**Comment by leo19941227** [comment]
Sorry I can't reproduce the issue Q
Can you execute the following command?

```
python3 run_downstream.py -m train -u fbank -d example -n test
```

I am using [this Colab](https://colab.research.google.com/drive/1VXE66_H2Vl92x6wR5tIqLPUa344Mm_jT?usp=sharing) for the above command.

**Comment by ashishBT** [comment]
I have executed the given cmd, but same error.
PS : I am using my system instead of Colab. have you tried to reproduce the error on your  system?

**Comment by leo19941227** [comment]
Yes I also ran on my system but did not encounter the error.
I paste the Colab link just to make sure that the environment is clean and the installation pipeline is reproducible.
Can you share the commit hash on master you are using?

**Comment by ashishBT** [comment]
Output of `git log` is:

```sh
commit ed9512709165bf4738817c04f5b529fc9bd7afc5 (HEAD -> master, origin/master, origin/HEAD)
Author: Leo Yang <leo19941227@gmail.com>
Date:   Tue Sep 21 10:27:01 2021 +0800

    refine ST doc

commit 726333e011f7ba548cbe415d2119a574b55ad173
Author: Leo Yang <leo19941227@gmail.com>
Date:   Tue Sep 21 09:56:05 2021 +0800

    refine SS

commit 5835a4fadadeed44fcbe7cf469e22b643c290f29
Merge: 1962724 d06d1ec
Author: Leo Yang <leo19941227@gmail.com>
Date:   Tue Sep 21 08:59:56 2021 +0800

    Merge branch 'master' of github.com:s3prl/s3prl

commit 196272426e22b9d53c70cab62994b1c9a381a501
Author: Leo Yang <leo19941227@gmail.com>
Date:   Tue Sep 21 08:59:47 2021 +0800

    centralized LibriMix

commit d06d1ec1091a57668688133b96f40f134ae6a72b
Author: Leo Yang <leo19941227@gmail.com>
Date:   Mon Sep 20 09:00:45 2021 +0800

    minor bug

commit 57348c08feab23835c97022654e64f7d3aafdaea
Author: Leo Yang <leo19941227@gmail.com>
Date:   Mon Sep 20 08:55:42 2021 +0800

    don't use find_namespace_package

commit 05d93154c90d61fd098cc08b6bdba8664d78a233
Author: Leo Yang <leo19941227@gmail.com>
Date:   Mon Sep 20 06:39:39 2021 +0800

:


```

**Comment by ashishBT** [comment]
This issue has been resolved after updating my python from 3.6 to 3.8.

**Comment by leo19941227** [comment]
Thanks for the information!
